test/i6pn_clustered_test.py

```diff
@@ -39,17 +39,6 @@ def test_experimental_group(servicer, client):
         assert fn3.i6pn_enabled is True
 
 
-def test_spawn_experimental_group(client, servicer, monkeypatch):
-    # We need to set a custom function body here since grouped function's kwargs
-    # aren't compatible with the default servicer function body.
-    @servicer.function_body
-    def grouped_func(*args, **kwargs):
-        return sum(args)
-
+def test_spawn_experimental_cluster(client, servicer, monkeypatch):
     with app.run(client=client):
-        # Needed for type checker to be happy. This isn't the best solution, but
-        # the feature is experimental anyway...
-        assert isinstance(f1, modal.experimental.GroupedFunction)
-        f1.client = client
-
-        assert f1.remote(2, 4) == [6] * 2
+        assert f1.remote(2, 4) == 2**2 + 4**2
```

Comment:
I'm confused where this result is coming from. Is it hardcoded into the mock servicer?

Reply:
Yeah. I'll add a comment